scripts/start/prepare_and_run.R

```diff
@@ -1173,6 +1173,11 @@ run <- function(start_subsequent_runs = TRUE) {
   # Postprocessing / Output Generation
   output    <- cfg$output
   outputdir <- cfg$results_folder
+
+  # make sure the renv used for the run is also used for generating output
+  stopifnot(is.null(renv::project()) || normalizePath(renv::project()) == normalizePath(outputdir))
```

Comment:
This is more of a documentation thing that should help with understanding the code, if this still fails I messed up. I added an error message anyway, just in case